torch/nn/modules/pixelshuffle.py

```diff
@@ -6,26 +6,30 @@
 
 class PixelShuffle(Module):
     r"""Rearranges elements in a tensor of shape :math:`(*, C \times r^2, H, W)`
-    to a tensor of shape :math:`(*, C, H \times r, W \times r)`.
+    to a tensor of shape :math:`(*, C, H \times r, W \times r)`, where r is an upscale factor.
```

Comment:
Using "an upscale factor" seems OK here since the name of the input is not available